### PR TITLE
#9 apply gitignore optimization

### PR DESCRIPTION
**Pull request by KoMinkyu**
PR created from #9

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$" vcs="Git" />
-  </component>
-</project>
```
Comment (poqw) [inline .idea/vcs.xml]: @KoMinkyu What exactly does this file do?

**Comment by poqw** [comment]
Do people typically create an issue first, then open a new branch to implement the feature and then implement the problem in the branch, and finally do the development by pushing the PR?

File: .idea/vcs.xml (hunk 1 above)
Comment (poqw) [inline .idea/vcs.xml]: .idea folder : 
> Project settings are stored with each specific project as a set of xml files under the .idea folder. If you specify the default project settings, these settings will be automatically used for each newly created project.

So configuration files that depend on the local environment need to be ignored. Thx.

**Comment by KoMinkyu** [comment]
@poqw nope. That makes spend more time to managing our issues.
Maybe we can discuss about, how to manage our issues. :)

**Pull request merged**